### DISCUSSION TITLE
Audio-only channels with visualizer and ID3 Tags on screen.

### DISCUSSION THREAD

**Discussion by CaliMiller** [chrisbenincasa/tunarr#219] [Ideas]
It'd be cool to create channels from our personal music collection or podcasts. Create channels based on genre, artist, or tags. Song name, artist, album name, album art, and other info pulled from ID3 tags, shown on screen.

Customizations:
Duration ID3 data stays on screen
Font size and text placement of ID3 tags
Scrolling or looping ID3 tags on screen
Customizable music visualizer... colors, waveform, effects
Optional custom background image behind visualization
Fade between songs
Filler option to add radio station bumpers, PSAs, advertisements, etc.
Customize Electronic Program Guide
Create programs based on genre, artist, etc for certain times of the day.

**Comment by chrisbenincasa** [comment]
Thanks for the suggestion! Going to move this over to discussions, since I think we're going to try and use that for feature requests going forward. That way the requests can be upvoted as well.
